_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Version 1
+---
+
+We are pleased to release version 1 of MDAnalysis!
+
+
+MDAnalysis 1.x is a stable legacy platform that supports Python 2.7, 3.5, 3.6, 3.7 and 3.8 on Windows (32- and 64-bit), MacOS, and Linux.
+The API will not change for any upcoming 1.x versions.
+This release completes the first phase of our [roadmap][].
+From here on, we will focus on developing version 2 of MDAnalysis, which will support Python 3.6+ only, and will include major API changes and improvements.
+
+
+The 1.0.0, 1.0.1 and 1.1.1 versions of MDAnalysis are the product of more than 18 months effort.
+They contain a multitude of **fixes**, [**deprecations**](#deprecations), and **new features**.
+We highlight important changes below, but users are strongly encouraged to read the [CHANGELOG][] for full details.
+All users are recommended to upgrade to 1.1.1, as it includes important fixes to neighbor searching, a fundamental component of many analyses in MDAnalysis.
+
+In [Dependencies](#dependencies) we list the required dependencies of MDAnaysis version 1.
+
+## GSoC? 
```

Comment:
@IAlibay not sure what you wanted to say here, aren't most of the changes in 2.0?

Reply:
This is following the 0.19.x format, the question here is "are there GSOC contributions we want to highlight". I guess here we are mainly thinking about GSOC 2019 or GSOD?